templates/header-right.mustache

```diff
@@ -3,11 +3,12 @@
 	  	<li class="dropdown">
 	    	<a href="http://bitovi.com" class="bitovi icon-bits">Bitovi</a>
 	    	<ul class="dropdown-menu">
-	      		<li><a href="http://bitovi.com">Bitovi.com</a></li>
-	      		<li><a href="http://bitovi.com/blog/">Blog</a></li>
-	      		<li><a href="http://bitovi.com/consulting/">Consulting</a></li>
-	      		<li><a href="http://bitovi.com/training/">Training</a></li>
-	      		<li><a href="http://bitovi.com/open-source/">Open Source</a></li>
+	      		<li><a href="https://bitovi.com">Bitovi.com</a></li>
+	      		<li><a href="https://bitovi.com/blog/">Blog</a></li>
+	      		<li><a href="https://bitovi.com/design/">Design</a></li>
+	      		<li><a href="https://bitovi.com/development/">Development</a></li>
+	      		<li><a href="https://bitovi.com/training/">Training</a></li>
+	      		<li><a href="https://bitovi.com/open-source/">Open Source</a></li>
```

Comment:
All of these redirect to a different URL, e.g. https://www.bitovi.com/open-source